packages/design-system/src/components/Icons/__snapshots__/SvgIcon.test.tsx.snap

```diff
@@ -39,7 +39,7 @@ exports[`SvgIcon wrapper icon can pass through additional props 1`] = `
   class="ds-c-icon ds-c-icon--add "
   data-testid="addIconTest"
   focusable="false"
-  id="icon-1"
+  id="icon--1"
```

Comment:
Was trying to figure out why this wasn't static, and then realized that it's an `<AddIcon>`. This would be a good place to test the pass-through of the id prop as well.